### PR TITLE
Fixed firmware datasource bug

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Fixed firmware datasource bug]

### Issues Resolved
[#523 #524 ]

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by alisha-k-kalladassery — APPROVED** [review]

File: oneview/data_source_firmware_drivers.go [hunk 1]
```diff
@@ -236,8 +233,10 @@ func dataSourceFirmwareDrivers() *schema.Resource {
 
 func dataSourceFirmwareDriversRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
-	id := d.Get("id").(string)
-	firmware, err := config.ovClient.GetFirmwareBaselineByNameandVersion(id)
+	name := d.Get("name").(string)
+	version := d.Get("version").(string)
+	id := name + version
```
Comment (AvijitOS) [inline oneview/data_source_firmware_drivers.go:238]: is there a possibility name comes with space? i mean id with space will not cause any issue?

Comment (nabhajit-ray) [reply oneview/data_source_firmware_drivers.go:238]: Name will can have space , but id is not used for for read.

File: oneview/data_source_firmware_drivers.go [hunk 2]
```diff
@@ -236,8 +233,10 @@ func dataSourceFirmwareDrivers() *schema.Resource {
 
 func dataSourceFirmwareDriversRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
-	id := d.Get("id").(string)
```
[the remark this answers is not in the document]
Comment (nabhajit-ray) [reply oneview/data_source_firmware_drivers.go:239]: Import is using the read operation of resource file and not datasource.

**Review by AvijitOS — APPROVED** [review]
hope u will be raising different defect to handle resource part

**Pull request merged**